ovsdb/client.go

```diff
@@ -105,6 +169,70 @@ func (c *Client) rpc(method string, out interface{}, args ...interface{}) error
 	return nil
 }
 
+// listen starts an RPC receive loop that can return RPC results to
+// clients via a callback.
+func (c *Client) listen() {
+	for {
+		res, err := c.c.Receive()
+		if err != nil {
+			// EOF or closed connection means time to stop serving.
+			if err == io.EOF || strings.Contains(err.Error(), "use of closed network") {
+				return
+			}
+
+			// For any other connection errors, just keep trying.
+			continue
```

Comment:
Not a show stopper but is there any way to focefully cancel this listen(), maybe passing `context.Context`?

Reply:
It's canceled by shutting down its network connection right now.  Was going to pass context but you can't cancel a listener with a context.

Yeah, I'm concerned about flapping logs.  May add more debug logging.

Reply:
:+1: